src/components/Editor.vue

```diff
@@ -490,6 +490,7 @@ export default {
 							this.$syncService.startSync()
 						},
 						onUpdate: ({ editor }) => {
+							// this.debugContent(editor)
```

Comment:
Very nice idea, we could always do this if the instance is in debug mode through the config.php option even (This is exposed as `window.oc_debug`)

Reply:
I fear that it might become confusing if you're debugging other stuff (not related to serialization) :wink:

Reply:
Also fine with me :)

Reply:
Still you have a point in making the debug function easier to discover. Maybe add a note to `README.md`?

Reply:
> Very nice idea, we could always do this if the instance is in debug mode through the config.php option even (This is exposed as `window.oc_debug`)

Or simply use `logger.debug` as that variable is already checked by the logger and if not set, the message is discarded.